Source/Letterbook.Web/wwwroot/css/app.css

```diff
@@ -1,145 +1,369 @@
 body.app {
     display: grid;
     grid-template-areas:
-        "header main" 
-        "footer main";
+        "header main";
     grid-template-columns: 1fr 3fr;
-    grid-template-rows: 1fr auto;
-    height: 100vh;
+    grid-template-rows: auto;
+    height: auto;
 
-    footer {
-        grid-area: footer;
+    & > header footer,
+    & > footer {
+        grid-area: header;
         font-size: 0.75rem;
+        position: fixed;
```

Comment:
need to figure out why making this sticky isn't working, because making if fixed also causes a bunch of problems